go/cli/mcap/utils/mcap_amendment.go

```diff
@@ -0,0 +1,454 @@
+package utils
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	"hash/crc32"
+	"io"
+
+	"github.com/foxglove/mcap/go/mcap"
+)
+
+const (
+	SIZE_RECORD_LENGTH = 8
```

Comment:
```suggestion
	SizeRecordLength = 8
```
seems more in the go idiom for package-level constants?